Lesson22/todo-app/aysel/script.js

```diff
@@ -0,0 +1,39 @@
+const form = document.getElementById('toDoForm');
+
+form.addEventListener('submit', handleSubmit);
+form.addEventListener('reset', handleReset);
```

Comment:
Reset button in the form should reset the input element(s). E.g. if there's some data in the input fields in the form, 'reset' button should clear all input fields. 
Current behaviour of resent button is incorrect and would be considered a misleading UI. Instead, remove this line but if you want a user to clear entire todo list by clicking a button, add a new button in HTML outside the form element for that.